openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
Okay, so I believe that you and @nasthagiri are basically having the same root concern. I'll make two sets of thumbnails, serve the smaller one on the dashboard and the larger one for the about pages. That being said, does it really just work when you put a double sized image? I don't really know how retina image scaling on web pages works, but I thought you had to specify something in terms of srcset or some such to get it to show up in high res mode.

FWIW, we do use the normal Open edX about and course listing pages our white label stuff (where the number of courses is relatively limited). And we are building a new Python based course discovery system. I didn't intend to downgrade the community on this -- most of the data I've seen (which I admit, is limited to our stuff) has course images that are either in the 375x200 range or fantastically beyond it, so I didn't think it would have a major impact on users to restrict it like that.